### PR TITLE
Ot139-94 Crear Seed de usuarios

### PR DESCRIPTION
**Pull request by Franzcod**
### Seed to Users

| firstname | lastname | email           | password | roleID   | image                           |
| --------- | -------- | --------------- | -------- | -------- | ------------------------------- |
| Usario1   | Demo1    | test1@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 1| https://via.placeholder.com/150 |
| Usario2   | Demo2    | test2@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 1| https://via.placeholder.com/150 |
| Usario3   | Demo3    | test3@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 1| https://via.placeholder.com/150 |
| Usario4   | Demo4    | test4@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 1| https://via.placeholder.com/150 |
| Usario5   | Demo5    | test5@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 1| https://via.placeholder.com/150 |
| Usario6   | Demo6    | test6@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |
| Usario6   | Demo6    | test6@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |
| Usario7   | Demo7    | test7@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |
| Usario8   | Demo8    | test8@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |
| Usario9   | Demo9    | test9@test.com  | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |
| Usario10  | Demo10   | test10@test.com | $2a$10$OYPdaBdxk8f8Vw20EwezKOA.hRNdgYJFtMYT3CJhHzRJOC4TpdutO| 2| https://via.placeholder.com/150 |

### CODE REVIEW

**Comment by Galaeno** [comment]
@Franzcod tenés conflictos

File: seeders/20220311013951-create-users.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        email: 'test10@test.com',
+        password: '1234',
+        roleId: 2,
+        image: 'https://via.placeholder.com/150',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+    ];
+
+    await queryInterface.bulkInsert('Users', users, {});
```
Comment (Galaeno) [inline seeders/20220311013951-create-users.js:108]: Estaría bueno confirmar con el equipo cuál va a ser el valor final.. si users o Users

Comment (Franzcod) [reply seeders/20220311013951-create-users.js:108]: Entiendo que quedo Users

**Comment by Galaeno** [comment]
@Franzcod tenés conflictos

**Review by Galaeno — APPROVED** [review]

**Pull request merged**